scripts/gulp-start-gae-devserver.js

```diff
@@ -0,0 +1,87 @@
+// Copyright 2015 The Oppia Authors. All Rights Reserved.
```

Comment:
I found the original repo this was based on: https://github.com/maciejzasada/gulp-gae

Why can't we just use it directly?